include/dlr_common.h

```diff
@@ -55,6 +54,9 @@ typedef struct {
 
 void ListDir(const std::string& dirname, std::vector<std::string>& paths);
 
+/* Logic to handle Windows drive letter */
+std::string pathFix(std::string path);
```

Comment:
Can you rename to something more descriptive like `FixWindowsDriveLetter`

Reply:
done